modules/claims_api/app/sidekiq/claims_api/claim_uploader.rb

```diff
@@ -48,14 +46,33 @@ def bd_upload_body(auto_claim:, file_body:, doc_type:)
       end
     end
 
-    def claim_bd_upload_document(claim, doc_type, pdf_path)
+    def claim_bd_upload_document(claim, doc_type, pdf_path) # rubocop:disable Metrics/MethodLength
       ClaimsApi::BD.new.upload(claim:, doc_type:, pdf_path:)
     # Temporary errors (returning HTML, connection timeout), retry call
     rescue Faraday::ParsingError, Faraday::TimeoutError => e
       ClaimsApi::Logger.log('benefits_documents',
                             retry: true,
                             detail: "/upload failure for claimId #{claim&.id}: #{e.message}; error class: #{e.class}.")
       raise e
+    # Check to determine if job should be retried based on status code
+    rescue ::Common::Exceptions::BackendServiceException => e
+      if will_retry_status_code?(e)
+        # rubocop:disable Layout/LineLength
+        ClaimsApi::Logger.log('benefits_documents',
+                              retry: true,
+                              detail: "/upload failure for claimId #{claim&.id}: #{e.message}; error class: #{e.class}.")
```

Comment:
I think it would be worth it to also port over the `get_error_message` method from the other base service and then for the message here use `get_error_message(e)`. That would provide more readable text and a better output into the logs.  (Plus then line 55 and 79 could also utilize it). For example with the example we used in the dev huddle it would return 

````
"Error calling external service to upload claim document."
````
Instead of the VA900 value that gets put n a lot of these exceptions.